There is a new object LastAddr.java for which the Main java file need…

PabloDeo opened the pull request:
…s to be accomodated. (See the new Main3.java for reference)

(I am not sure how I should modify the Main2.java .)

Commit:
There is a new object LastAddr.java for which the Main java file needs to be accomodated. (See the new Main3.java for reference)

Commit:
A small correction for border condition

joov reviewed (changes requested):
Dear Yanghua,
as for the block hashes (starting / ending)  and configurability I still have questions.
As you can see in Main2.java I pushed the curr and last hashes are read from a property file. From the code it is not obvious, if the values from Main2.java or from Main3.java are used. Could you please:
- delete the  fields not used
- use the the way the values are read from BCProperties as specified in Main2.java 

Again I would like to emphesize, that the code should be runnable *without* code changes (changes in .java-files) when restarting or running in another environment than your PC.

PabloDeo commented:
Dear Johannes,

I have just pushed the newer version.
To avoid confusion. I deleted the file Main2.java and altered Main3.java in such a way that it is working like Main2.java.

Best regards,
Yanghua

Commit:
Dear Johannes,
I did the change in terms of my best understanding of the procedure regarding the property files. (In the Main2.java from you, there was a line BCProperties.getProperty("last.addr", "12hLCoGF2bRr7TmFeS3Pr5zASYqfRXoKgj"). However, I could not find anything about last.addr in the property file. So I am unsure where does last.addr come from. I just added last.addr into the property file, along with multiple others.) The reason why the csv files you obtained are almost the same is that the parameter in the main files are never updated by the parser (as it always update the parameter in Main3), sorry that I overlooked it.

Best regards,
Yanghua

Commit:
It seems that address nodes could not be properly deleted as there are relationships attached to it.

joov reviewed (approved):
Changes approved
